docs/rollkit-stack.md

```diff
@@ -1,30 +1,26 @@
 ---
 sidebar_position: 3
 sidebar_label: Rollkit Stack
-description: Intro to Rollkit's stack, a modular development kit for building rollup chains.
+description: Intro to Rollkit's stack, a modular rollup framework.
 ---
 
 # The Rollkit Stack
 
 This section will cover the technical stack of Rollkit.
 
-Rollkit is built by replacing Tendermint, the Cosmos-SDK Consensus Layer,
```

Comment:
why remove this? It seems like important context for the rollkit stack

Reply:
(also as mentioned in my google doc, it's not accurate to refer to Tendermint as "the Cosmos consensus layer")

Reply:
It was duplicated in the intro and rollkit stack